sources/megaproxylist.go

```diff
@@ -0,0 +1,82 @@
+package sources
+
+import (
+	"bytes"
+	"context"
+	"encoding/csv"
+	"fmt"
+	"io"
+	"net/http"
+	"time"
+
+	"github.com/nfx/slrp/pmux"
+	"github.com/rs/zerolog/log"
+)
+
+func init() {
+	Sources = append(Sources, Source{
+		ID:        65,
+		Homepage:  "https://www.megaproxylist.net",
+		Frequency: 24 * time.Hour,
+		Seed:      true,
+		Feed:      simpleGen(Megaproxylist),
+	})
+}
+
+type megaproxylistLine struct {
+	address   string
+	port      string
+	country   string
+	reability string
+}
+
+var megaproxylistUrl = fmt.Sprintf("https://www.megaproxylist.net/download/megaproxylist-csv-%s_SDACH.zip", time.Now().Format("20060102"))
```

Comment:
What is SDACH suffix, by the way? Leave some comments above this line

Reply:
is just their suffix that is static

Reply:
is just their suffix that is static